src/compiler/scale.ts

```diff
@@ -364,54 +365,55 @@ scale.color = function(encoding, name, scaleType, stats) {
   }
 };
 
-scale.color.palette = function(range, cardinality, type) {
-  // FIXME(kanitw): Jul 29, 2015 - check range is string
-  switch (range) {
-    case 'category10k':
-      // tableau's category 10, ordered by perceptual kernel study results
-      // https://github.com/uwdata/perceptual-kernels
-      return ['#2ca02c', '#e377c2', '#7f7f7f', '#17becf', '#8c564b', '#d62728', '#bcbd22', '#9467bd', '#ff7f0e', '#1f77b4'];
+export var colors = {
```

Comment:
Kind of. I'm sure there are better ways to do this but I don't want to spend time on it until we decide to go with it.